lte/gateway/python/magma/pipelined/tests/test_ipv6_solicitation_5g.py

```diff
@@ -0,0 +1,185 @@
+    OTHER_MAC = '0a:00:27:00:00:02'
+    OTHER_IP = '1.2.3.4'
+
+    @classmethod
+    def setUpClass(cls, *_):
+        """
+        Starts the thread which launches ryu apps
+
+        Create a testing bridge, add a port, setup the port interfaces. Then
+        launch the ryu apps for testing pipelined. Gets the references
+        to apps launched by using futures.
+        """
+        super(IPV6RouterSolicitation5GTableTest, cls).setUpClass()
+        warnings.simplefilter('ignore')
+        cls.service_manager = create_service_manager(
+            [],
+            ['ipv6_solicitation'],
+        )
+        cls._tbl_num = cls.service_manager.get_table_num(IPV6SolicitationController.APP_NAME)
+
+        ipv6_controller_reference = Future()
+        testing_controller_reference = Future()
+        test_setup = TestSetup(
+            apps=[
+                PipelinedController.IPV6RouterSolicitation,
+                PipelinedController.Testing,
+                PipelinedController.StartupFlows,
+            ],
+            references={
+                PipelinedController.IPV6RouterSolicitation:
+                    ipv6_controller_reference,
+                PipelinedController.Testing:
+                    testing_controller_reference,
+                PipelinedController.StartupFlows:
+                    Future(),
+            },
+            config={
+                'setup_type': 'LTE',
+                'allow_unknown_arps': False,
+                'bridge_name': cls.BRIDGE,
+                'bridge_ip_address': cls.BRIDGE_IP,
+                'ovs_gtp_port_number': 32768,
+                'virtual_interface': cls.BRIDGE,
+                'local_ue_eth_addr': True,
+                'quota_check_ip': '1.2.3.4',
+                'ipv6_router_addr': 'd88d:aba4:472f:fc95:7e7d:8457:5301:ebce',
+                'clean_restart': True,
+                'virtual_mac': 'd6:34:bc:81:5d:40',
+                'enable_nat': True,
+                'classifier_controller_id': 5,
+                'enable5g_features': True,
+            },
+            mconfig=PipelineD(
+                ue_ip_block=cls.UE_BLOCK,
+            ),
+            loop=None,
+            service_manager=cls.service_manager,
+            integ_test=False,
+        )
+
+        BridgeTools.create_bridge(cls.BRIDGE, cls.IFACE)
+
+        cls.thread = start_ryu_app_thread(test_setup)
+        cls.solicitation_controller = ipv6_controller_reference.result()
+        cls.testing_controller = testing_controller_reference.result()
+
+        cls._prefix_dict = {}
+        cls.solicitation_controller._prefix_mapper._prefix_by_interface = \
+            cls._prefix_dict
+
+    @classmethod
+    def tearDownClass(cls):
+        stop_ryu_app_thread(cls.thread)
+        BridgeTools.destroy_bridge(cls.BRIDGE)
+
+    def test_ipv6_5g_flows(self):
+        """
+        Verify that a UPLINK->UE Router Solic & Router Adv
+        """
+        ll_addr = get_if_hwaddr('testing_br')
```

Comment:
starting here, down to line 164 does not seem to be needed for the test. Also respective imports and class variables.

Reply:
@prabinakpattnaik : can you answer here?

Reply:
the variables `pkt_rs` and `pkt_ns` are used in lines 179f. It also registers a package injector on a interface defined in setup, that is used later. For me that code looks needed.

Reply:
Here is a simplified version that is green for me: https://github.com/magma/magma/commit/5bc798c17430a6abcbb7ae0c7ce2b053aa0fc7f1

my understanding: the tests uses `include_stats=False`, i.e., it does not matter if any packages are sent because they are not verified in the rule statistics.

Reply:
ok, so this test does not test that much.

Reply:
The same is true for the test you copied the code from, due to the following line https://github.com/magma/magma/blob/a6d697122fba15843129109f3410f89a0e9477f8/lte/gateway/python/magma/pipelined/tests/test_ipv6_solicitation.py#L173

I think this needs a follow-up ticket where we resolve the tests which have unused code / test less then expected. I created https://github.com/magma/magma/issues/13616 for that

Reply:
I have removed the ```include_stats=False,``` for IPv6 traffic verification along with flows. It's not a dead code.

Reply:
In the non 5g case this was set to False because of test instability (see respective PR). I did run the test locally 10 times -> all green. But this is of course no guarantee for CI.

Reply:
@prabinakpattnaik example for the instability issue in CI for the 5g case: https://github.com/magma/magma/runs/7907532515?check_suite_focus=true